### PR TITLE
Feat: Added two fields for address and registration date

### PR DESCRIPTION
**Pull request by Hardik-2311**

### CODE REVIEW

**Comment by Hardik-2311** [comment]
@beingnoble03

File: src/components/register_student/index.js [hunk 1]
```diff
@@ -562,6 +580,26 @@ class RegisterStudent extends React.Component {
...
+                <Form.Field>
+                  <label>"Address (As Per Bhawan)"</label>
+                  <Input
+                    name='address_bhawan'
+                    value={address_bhawan}
+                    onChange={this.fieldsChange}
+                    loading={loading}
+                  />
+                </Form.Field>
+                <Form.Field fluid widths='equal'>
```
Comment (beingnoble03) [inline src/components/register_student/index.js:593]:
```suggestion
                <Form.Field>
```

@Hardik-2311 I think this should be changed?

Comment (Hardik-2311) [reply src/components/register_student/index.js:593]: apne hi kaha tha input field match krni hai bkiyo se vohi hai yeh

Comment (beingnoble03) [reply src/components/register_student/index.js:593]: it's required for `Form.Group`, not for `Form.Field` i think.

**Comment by Hardik-2311** [comment]
Done

**Pull request merged**